src/components/Sidebar.js

```diff
@@ -115,11 +120,8 @@ const Sidebar = ({
         <h2 id="heading-add">
           <VisuallyHidden>Add a Loo</VisuallyHidden>
         </h2>
-        <Box
-          as={NavLink}
+        <StyledNavLink
```

Comment:
out of interest, was `as={NavLink}` causing problems or was it changed due to preference?

Reply:
it was causing problems, the alignItems prop was being applied as an attribute and my browser was complaing